react-ui/src/App.jsx

```diff
@@ -1,53 +1,34 @@
 /**
  * Entry-point into the website. Contains theme and routing info.
+ * Modal: https://v5.reactrouter.com/web/example/modal-gallery
+ * Are entryIds unique?
```

Comment:
Yes, but there is a many-to-many relationship between contests and entries.

Reply:
I feel like this is the third time I've asked this lol, I think I just forgot to remove this line